scopes.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Access scopes
+---
+
+diaspora* API features access scopes (or just 'scopes') in order to implement resource access management for API clients.
+
+Scopes are part of [OAuth 2.0 specification][oauth2] which is a base for [OpenID Connect Core 1.0][connect] implemented by diaspora*.
+
+When the API client requests an access token it sends the list of scopes it wants to get access to accoring to the selected authorization flow (see ["Authentication"][authentication]).
+
+Approved scopes may differ from what was requested by the client.
+
+[OpenID Connect Core 1.0][connect] defines a few scopes which are specific for getting the authorized user information. API client must include the `openid` scope to the scope request list. `openid` scope means "allow OpenID Connect authentication" and if missing then API authentication will fail.
+
+diaspora* also supports `profile` and `email` scopes according to [OpenID Connect Core 1.0][connect-scopes].
+
+Other scopes are diaspora* API specific. All scopes besides `openid` are optional.
+
+Each API endpoint requires at least one scope to be approved. If the access token used for making a request wasn't approved for the scope of the respective endpoint then `403` status will be responded.
```

Comment:
"approved" -> "granted"
"approved for the" -> "granted the"
"then it will respond with a `403` status code."

Reply:
> "then it will respond with a 403 status code."

Maybe "then **the endpoint** will respond with a 403 status code" is better?

Reply:
well, then it would be "of the respective endpoint then the endpoint" which is kinda repetitive. We could also say "then the response will have a `403` status code and no actual content".